test/lib/controller/test.js

```diff
@@ -0,0 +1,26 @@
+const chai = require("chai");
```

Comment:
The convention you are using is to create a test module named `test.js` for a ES6 module named `index.js`. To be consistent with the naming scheme, it is better to call the test module as `index.js`.